### PR TITLE
[Improvement] accelerate T5 model conversion and fix bloom model on multi-process

### PR DESCRIPTION
**Pull request by lanking520**
For example,

Flan-T5-xxl (49GB) conversion time reduced from 11 min to 4 min on a 64 core CPU with 4 concurrent process

If you do:

```
-i bigscience/bloomz-3b -o /tmp/ft_model3/ -tp 1 -p 4 -dt fp32
```

With Bloom conversion script. You will reproduce the bus error. This PR include a fix to address that

### CODE REVIEW

**Comment by lanking520** [comment]
@byshiue

**Review by rohithkrn — APPROVED** [review]

File: examples/pytorch/gpt/utils/huggingface_bloom_convert.py [hunk 1]
```diff
@@ -209,9 +209,8 @@ def handle_exceptions(model_config: PretrainedConfig,
     return param
 
 
-def convert_and_save_parameter(config: PretrainedConfig,
-                               name: str,
-                               param: torch.nn.Parameter,
+def convert_and_save_parameter(param_name: str,
```
Comment (byshiue) [inline examples/pytorch/gpt/utils/huggingface_bloom_convert.py:212]: You change the API, but don't update the line 333.

Comment (lanking520) [reply examples/pytorch/gpt/utils/huggingface_bloom_convert.py:212]: I changed that in the following commit. But maybe we can just remove if else statement and use star_async. 1 process can still work under this condition.

Comment (lanking520) [reply examples/pytorch/gpt/utils/huggingface_bloom_convert.py:212]: I would assume everyone even the laptop now should have 4 cores, so 1 core condition not common

**Pull request merged**